sips/sip-018/sip-018-signed-structured-data.md

```diff
@@ -0,0 +1,204 @@
+# Preamble
+
+SIP Number: 018
+
+Title: Signed Structured Data
+
+Author: Marvin Janssen <https://github.com/MarvinJanssen>
+
+Consideration: Technical
+
+Type: Standard
+
+Status: Accepted
+
+Created: 28 December 2021
+
+License: CC0-1.0
+
+Sign-off: Jude Nelson <jude@stacks.org>, Technical Steering Committee Chair
+
+Layer: Applications
+
+Discussions-To: https://github.com/stacksgov/sips
+
+# Abstract
+
+The Signed Structured Data specification describes a standard way to deterministically encode and sign Structured Data. Structured Data is data that can be represented in human-readable format and used in applications and smart contracts. The aim of the standard is to produce signatures that are straightforward and inexpensive to verify by smart contracts.
+
+# License and Copyright
+
+This SIP is made available under the terms of the Creative Commons CC0 1.0 Universal license, available at https://creativecommons.org/publicdomain/zero/1.0/
+This SIP's copyright is held by the Stacks Open Internet Foundation.
+
+
+# Introduction
+
+Digital signatures are at the heart of blockchains. They allow users to trustlessly transfer assets, invoke smart contracts and more. To perform these actions, a user signs a transaction and broadcasts it to the network. However, there are situations in which signed proofs without having to broadcast transactions are desirable. A few common use-cases include: (1) prove to an external application or entity that a user is in control of an address; (2) authorise an action to be performed by a smart contract at a later stage (like a meta transaction); (3) participate in an off-chain mechanism that is later settled on-chain (like a subnet).
```

Comment:
A few things here:

* "Trustlessly" isn't correct -- there is definitely some trust involved in a blockchain's operation.  Can we say "they allow users to transfer assets, invoke smart contracts, and more, without a designated trusted third party," which is more correct?

* "Signed proofs" isn't a term I'm familiar with.  Maybe it would be better to say "there are situations in which it is desirable to submit signed data to a blockchain without encoding it as a transaction"?

* Formatting: Can you separate these list items with newlines?

* What's a "meta transaction?" This is the first time it's been used in this document.

Reply:
1. Sure, that is what I meant.
2. Sure. But this data is not necessarily meant to be submitted to a blockchain. For example, many dapps in the EVM-space and beyond authenticate users on centralised components using signed messages.
3. Ok.
4. Added under "Definitions".